vision_stack/buoy_detect/README.md

```diff
@@ -0,0 +1,24 @@
+BUOY_DETECT
+-----------
+This repository contains the codes that uses ML (Machine Learning) techniques to detect the buoy. This is a combination of 3 parts - 
+<br>
```

Comment:
I guess you've tagged me @pranaypratyush :wink: 

Sure, @icyflame - that'll be taken care of, from next time. :smile: